### ISSUE TITLE
Time counter incorrect on matrix builds

### ISSUE THREAD

**Issue by smiller171** [travis-ci/travis-ci#4728]
It appears that your time counter is counting correctly until a build finishes, at which point if it is a matrix build, the time counter adds up the individual runtimes instead of giving the real time that all of them took in parallel

**Comment by BanzaiMan** [comment]
I think this is a duplicate of #1486.

**Comment by joshk** [comment]
Thanks for the report @smiller171. The time counter isn't incorrect, it is just a different way to show the build time.

This is a duplicate of the issue mentioned by Hiro, so I'm going to close this for now.

Thanks a bundle.
